src/gc/gc.cpp

```diff
@@ -10442,7 +10442,7 @@ gc_heap::init_gc_heap (int  h_number)
     make_background_mark_stack (b_arr);
 #endif //BACKGROUND_GC
 
-    adjust_ephemeral_limits();
+    adjust_ephemeral_limits(true);
```

Comment:
This is up to you, but in cases like this where it's not immediately clear what the parameter means and the function declaration isn't nearby, I'd do something like  `adjust_ephemeral_limits(true /* is_runtime_suspended */);`